nifi-nar-bundles/nifi-framework-bundle/nifi-framework/nifi-framework-components/src/test/java/org/apache/nifi/processor/TestSimpleProcessLogger.java

```diff
@@ -24,16 +32,14 @@
 import static org.mockito.Mockito.verify;
 import static org.mockito.Mockito.when;
 
-import java.lang.reflect.Field;
+public class TestSimpleProcessLogger {
 
-import org.apache.nifi.logging.LogLevel;
-import org.apache.nifi.reporting.ReportingTask;
-import org.junit.Before;
-import org.junit.Test;
-import org.slf4j.Logger;
+    private static final String NEW_LINE_ARROW = "\u2517\u25B6";
+    private static final String EXPECTED_CAUSES = "java.lang.RuntimeException: first" + System.lineSeparator() +
```

Comment:
The order of the expected exceptions should be: "third", "second", "first". (Note that "third" is the root cause.)
See my comment on `SimpleProcessLogger`.